### PR TITLE
build: automate build & launch of storybook

### PR DESCRIPTION
**Pull request by andreaslordos**
🏆 Enhancements
- Added `build.py` in the `/packages/superset-ui-plugins-demo` directory
	- Running `build.py` will build & start storybook automatically
 	- No need to run 4 separate commands to do so
 

📜 Documentation
 - Modified README.md to reflect this change and provide instructions on using `build.py`

### CODE REVIEW

**Review by etr2460 — CHANGES_REQUESTED** [review]
cool idea! Although I think the more standard way would be to add a command in the demo `package.json` file. Then you could just use bash as opposed to python to do `cd ../../ && yarn build && cd - && yarn storybook`

**Comment by kristw** [comment]
You also only need to build once. The subsequent storybook calls should not need to rebuild

**Comment by andreaslordos** [comment]
> You also only need to build once. The subsequent storybook calls should not need to rebuild

@kristw I believe if you edit the code of a plugin/add your own, you will need to build Storybook again for the changes to take effect.

**Comment by kristw** [comment]
@andreaslordos You can deep import from `@superset-ui/xxx/src/yyy` instead of `@superset-ui/xxx` (which will point to the built files) during development.

**Comment by andreaslordos** [comment]
> cool idea! Although I think the more standard way would be to add a command in the demo `package.json` file. Then you could just use bash as opposed to python to do `cd ../../ && yarn build && cd - && yarn storybook`

@etr2460 thank you for the suggestion! I have modified my PR accordingly

File: packages/superset-ui-plugins-demo/package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "demo:build": "build-storybook -o _gh-pages",
     "demo:publish": "gh-pages -d _gh-pages",
     "deploy-demo": "npm run demo:clean && npm run demo:build && npm run demo:publish && npm run demo:clean",
-    "storybook": "start-storybook -p 9001"
+    "storybook": "start-storybook -p 9001",
+    "build": "cd ../../ && yarn build && cd - && yarn storybook"
```
Comment (etr2460) [inline packages/superset-ui-plugins-demo/package.json]: Much better! Maybe one final nit, I'd name this something like `build-and-run` or maybe follow the example with `demo`, naming the old `storybook` command `storybook:run` and this one just `storybook`

**Review by etr2460 — COMMENTED** [review]
Make sure you address the commit lint issues too (format the commit title properly)

File: packages/superset-ui-plugins-demo/package.json (hunk 1 above)
Comment (andreaslordos) [reply packages/superset-ui-plugins-demo/package.json]: Addressed commit lint issues (I think) & renamed the script name to "storybook" - thank you for the suggestions!

File: README.md [hunk 2]
```diff
@@ -106,7 +106,13 @@ You can demo your changes by running the storybook demo locally with the followi
...
 yarn build
 cd packages/superset-ui-plugins-demo
-yarn storybook
+yarn storybook:run
+```
+
+Alternatively, you can demo your changes by using the following command while in `packages/superset-ui-plugins-demo`:
+
+```sh
+npm run storybook
```
Comment (etr2460) [inline README.md]: one final nit, this should be `yarn storybook`

**Review by etr2460 — CHANGES_REQUESTED** [review]
almost there!

File: README.md (hunk 2 above)
Comment (andreaslordos) [reply README.md]: @etr2460 fixed

**Comment by andreaslordos** [comment]
Requesting review @etr2460

**Review by etr2460 — APPROVED** [review]
lgtm, thanks for the contribution!

**Pull request merged**